packages/jest-runner/src/index.ts

```diff
@@ -49,7 +49,7 @@ namespace TestRunner {
 class TestRunner {
   private _globalConfig: Config.GlobalConfig;
   private _context: JestTestRunnerContext;
-  private UNSTABLE_eventEmitter = new Emittery.Typed<JestTestEvents>();
+  public UNSTABLE_eventEmitter = new Emittery.Typed<JestTestEvents>();
```

Comment:
I think what Simen meant was to keep the scope `private` and remove the `UNSTABLE_` prefix

Reply:
yes 🙂 Will need an `on` method on the class or some such as it's accessed from `TestScheduler`.

```ts
on = this.UNSTABLE_eventEmitter.on.bind(this.UNSTABLE_eventEmitter);
```

seems to work locally for me

Reply:
Ah my bad, got it 👍

Reply:
not sure if the `bind` is necessary - try without it and if it works leave it out